waku/v2/node/wakunode2.nim

```diff
@@ -83,7 +83,7 @@ proc removeContentFilters(filters: var Filters, contentFilters: seq[ContentFilte
   # Flatten all unsubscribe topics into single seq
   var unsubscribeTopics: seq[ContentTopic]
   for cf in contentFilters:
-    unsubscribeTopics = unsubscribeTopics.concat(cf.contentTopics)
+    unsubscribeTopics = unsubscribeTopics.concat(@[cf.contentTopic])
```

Comment:
See comment on similar logic in `wakufilter`. This can now be replaced by a simple `mapIt` statement.